renovate.json

```diff
@@ -2,5 +2,12 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "ignoreDeps": [
+        "k8s.io/*"
```

Comment:
Error type: The renovate configuration file contains some invalid settings
Message: packageRules[0]: Each packageRule must contain at least one match* or exclude* selector. Rule: {"ignoreDeps":["k8s.io/*"]}

@h3poteto  looks like ignoreDeps should be outside of packageRules

Reply:
wow

Reply:
how did find that error message?

Reply:
https://github.com/oviceinc/rollout-optimizer-controller/issues/15#event-15415499948

Reply:
oh